src/seedu/addressbook/commands/HelpCommand.java

```diff
@@ -1,29 +1,30 @@
 package seedu.addressbook.commands;
 
-import static seedu.addressbook.TextUi.LS;
 
 /**
  * Format full help instructions for every command for display.
  */
-public class HelpCommand implements Command {
+public class HelpCommand extends Command {
 
     public static final String COMMAND_WORD = "help";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions."
-            + LS + "Example: " + COMMAND_WORD;
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
+            + "Example: " + COMMAND_WORD;
 
     public HelpCommand() {}
 
     @Override
-    public String execute() {
-        return AddPersonCommand.MESSAGE_USAGE
-                + LS + DeletePersonCommand.MESSAGE_USAGE
-                + LS + ClearAddressBookCommand.MESSAGE_USAGE
-                + LS + FindPersonsByWordsInNameCommand.MESSAGE_USAGE
-                + LS + ListAllPersonsCommand.MESSAGE_USAGE
-                + LS + ViewPersonDetailsCommand.MESSAGE_USAGE
-                + LS + ViewAllPersonDetailsCommand.MESSAGE_USAGE
-                + LS + HelpCommand.MESSAGE_USAGE
-                + LS + ExitCommand.MESSAGE_USAGE;
+    public CommandResult execute() {
+        return new CommandResult(
+                AddPersonCommand.MESSAGE_USAGE
+                + "\n" + DeletePersonCommand.MESSAGE_USAGE
+                + "\n" + ClearAddressBookCommand.MESSAGE_USAGE
+                + "\n" + FindPersonsByWordsInNameCommand.MESSAGE_USAGE
+                + "\n" + ListAllPersonsCommand.MESSAGE_USAGE
+                + "\n" + ViewPersonDetailsCommand.MESSAGE_USAGE
+                + "\n" + ViewAllPersonDetailsCommand.MESSAGE_USAGE
+                + "\n" + HelpCommand.MESSAGE_USAGE
+                + "\n" + ExitCommand.MESSAGE_USAGE
```

Comment:
`\n` is replaced by the correct formatting before displaying to the user. Formatting should only happen in the TextUi.